addons/binding/org.openhab.binding.miele/ESH-INF/thing/xgw3000.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Miele Bridge -->
+    <bridge-type id="xgw3000">
+        <label>Miele Bridge</label>
+        <description>The miele bridge represents the Miele@Home XGW3000 gateway.</description>
+
+        <properties>
+        	<property name="vendor">Miele</property>
+        </properties>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of the Miele@Home gateway.</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="interface" type="text">
+                <context>network_address</context>
+                <label>Network Address of the Multicast interface</label>
+                <description>Network address of openHAB host interface where the bindign will listen for multicast events coming from the Miele@Home gateway</description>
+                <required>true</required>
```

Comment:
Ah, sorry - the address must be the one of the server running openHAB of course.
But anyhow, could this be made optional and the ip address of the first network interface found being used as a default?
Btw, when entering an invalid ip, the binding ends up consuming 100% CPU with millions of messages like
```
2016-08-29 14:16:24.245 [ERROR] [b.m.handler.MieleBridgeHandler:364  ] - An exception occurred while receiving multicast packets : 'bad argument for IP_MULTICAST_IF: address not bound to any interface'
2016-08-29 14:16:24.245 [ERROR] [b.m.handler.MieleBridgeHandler:374  ] - An exception occurred while leaving multicast group : 'Can't assign requested address'
2016-08-29 14:16:24.246 [ERROR] [b.m.handler.MieleBridgeHandler:364  ] - An exception occurred while receiving multicast packets : 'bad argument for IP_MULTICAST_IF: address not bound to any interface'
```